From: Ben Westover <me@benthetechguy.net>
CMake is hard coded to look for libicu version 56, since their Windows
install of Qt6 is bundled with it. Since we're using system Qt, we can
use the system version of libicu.
---

diff --git a/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake b/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
index 10e1ea1..53a336a 100644
--- a/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
+++ b/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
@@ -25,6 +25,7 @@ set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_BINDIR}" "${CMAKE_INSTALL_LIBDIR}" "."
 install(DIRECTORY "${QT_DIR}/qml" "${QT_DIR}/plugins"
         DESTINATION "${CMAKE_INSTALL_PREFIX}")
 
+find_package(ICU 56.0 COMPONENTS i18n uc data REQUIRED)
 macro( AppendLib LIB_NAME HINT_PATH)
     string(TOUPPER ${LIB_NAME} UP_NAME)
 
@@ -54,9 +55,9 @@ AppendQt6Lib("libQt6Gui.so.6")
 AppendQt6Lib("libQt6Core.so.6")
 AppendQt6Lib("libQt6QuickTemplates2.so.6")
 AppendQt6Lib("libQt6DBus.so.6")
-AppendQt6Lib("libicui18n.so.56")
-AppendQt6Lib("libicuuc.so.56")
-AppendQt6Lib("libicudata.so.56")
+AppendQt6Lib("libicui18n.so")
+AppendQt6Lib("libicuuc.so")
+AppendQt6Lib("libicudata.so")
 AppendQt6Lib("libQt6XcbQpa.so.6")
 AppendQt6Lib("libQt6WaylandClient.so.6")
 AppendQt6Lib("libQt6WlShellIntegration.so.6")
